modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/PostalAddressResourceImpl.java

```diff
@@ -61,6 +64,25 @@ public void deletePostalAddress(Long postalAddressId) throws Exception {
 		_addressService.deleteAddress(postalAddressId);
 	}
 
+	@Override
+	public void deletePostalAddressByExternalReferenceCode(
+			String postalAddressExternalReferenceCode)
+		throws Exception {
+
+		Address address =
+			_addressLocalService.getAddressByExternalReferenceCode(
```

Comment:
I'm crying for this 😢  ... not your fault but it's wrong ... it should be a fetch, not a get.
We need to ask to @riccardo-alberti what we want to do... change "everything" or cross finger